packages/react-components/react-overflow/src/Overflow.cy.tsx

```diff
@@ -19,25 +20,36 @@ const selectors = {
   menu: 'data-test-menu',
 };
 
-const Container: React.FC<{ children?: React.ReactNode; width?: number } & Omit<OverflowProps, 'children'>> = ({
-  children,
-  width,
-  ...overflowProps
-}) => {
+const Container: React.FC<
+  { children?: React.ReactNode; width?: number; overflowAxis?: OverflowAxis } & Omit<OverflowProps, 'children'>
+> = ({ children, width, ...userProps }) => {
   const selector = {
     [selectors.container]: '',
   };
 
+  const overflowProps = {
+    overflowAxis: 'horizontal' as const,
```

Comment:
Let's just do:

```
{
  children,
  overflowAxis = 'horizontal',
  width,
  ...userProps
}
```

- It removes `overflowProps`
- It simplifies `resize: overflowProps.overflowAxis ?? 'horizontal',` to `resize: overflowAxis,`